metadata-ingestion/src/datahub/ingestion/source/sql/trino.py

```diff
@@ -42,6 +56,21 @@
 register_custom_type(datatype.MAP, MapTypeClass)
 register_custom_type(datatype.DOUBLE, NumberTypeClass)
 
+
+@dataclass
+class PlatformDetail:
+    platform_name: str
+    is_three_tier: bool
+
+
+KNOWN_CONNECTOR_PLATFORM_MAPPING = {
```

Comment:
we should have two things 
- a mapping of trino connector names -> datahub platform names
- a list of datahub platform names that are two-tier

I don't think we need the PlatformDetail thing